lib/devtools/rake/rubocop.rb

```diff
@@ -0,0 +1,25 @@
+module Devtools
+  module Rake
+    # Rubocop metric runner
+    class Rubocop
+      include Anima.new(:config_file, :directory), Procto.call(:verify)
+
+      # Verify rubocop integration runs successfully
+      #
+      # rubocop:disable Style/RedundantBegin
```

Comment:
I agree. I decided to extract before refactoring since that is where I am most likely to lose momentum